_posts/2018-07-31-Agile-Leadership-Game.md

```diff
@@ -0,0 +1,64 @@
+
+<img alt="Agile Leadership Game" src="{{ '/img/agile-leadership-game/Agile-Leadership-Game.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 750px;">
+
+If the characteristic on the card is behaviour that the organisation DOES HAVE at that time but that you DON’T WANT, the card gets sorted in the quadrant “LET GO”.
+If the participant perceives the characteristic on the card as a behaviour the organisation DOES HAVE and that you DO WANT, it ends up at “KEEP” and so on.
+An example of a characteristic is “management spends a lot of time putting out fires”.
+
+There is no right or wrong: the game provides insight in the current state of affairs.
+The characteristics are labelled in types of behaviour belonging to types of organisations.
+So you could end up with lots of characteristics belonging to agile organisations in the “CREATE” quadrant. :)
+
+After the time box for sorting the cards ends, the participants will discuss the characteristics that seem most relevant to them and their organisation.
+Each participant gets the chance to enter at least one card/characteristic into the discussion.
+This needs to be (well) facilitated by an experienced coach/agile master.
+You can take as long as you like for the discussion and could do several discussion rounds, but we would recommend not to have it last for too long (1.5 hours max).
+
+After the discussion round it is useful to add an extra step in the game as to make the outcome more actionable:
+the participants prioritise the top three of characteristics/topics they would like to let go or create.
+These are the topics they can start working with the very next day!
+This way you also prevent the session being perceived as just fun, irrelevant and/or without consequences.
+Because there are forty two cards in the game and you won’t discuss them all, the game is most suitable to be played again at a later time – it won’t lose its value.
+
+### Outcomes retrospective:
+
+**TIP:**
+* Participants need some time to figure out how to play the game and how to work with the quadrants on the board.
+* An ice breaker to start with and a small warm up round increases effectiveness and actionable outcome of the game.
+* Making sure the topics in the discussion round are precisely those that matter most to the organisation and the participants, adds greatly to the usefulness of the session.
+* You need a skilled facilitator.
+
+**TOP:**
+* Very nice game to play, offering lots of insight and actionable openings to start your transformation at leadership level.
+* The game doesn't take too long, can easily be played several times over time and can easily be fitted into a leadership programme, workshop, curriculum or “heidag”.
+* It’s a perfect self assessment game for management teams that are wondering how agile they actually are as a team. :)
+
+
+More information about the game can be found [here](https://www.agileconsortium.net/leadership-game/){:target="_blank"}.
```

Comment:
Seeing as you mention multiple times that it is very important to have a good facilitator, you could mention something about reaching out if the reader would be interested to have a good facilitator help them? :-)

Reply:
Ah, good idea!